### PR TITLE
fix(RSVP): Check for block content and don't show classic form if present.

### PR DESCRIPTION
**Pull request by Camwyn**
Test the post content for blocks and for the tribe-block class before allowing the RSVP classic form to display.

:ticket: https://central.tri.be/issues/124394

### CODE REVIEW

File: src/Tribe/RSVP.php [hunk 1]
```diff
@@ -1071,18 +1071,13 @@ public function front_end_tickets_form( $content ) {
...
-
+		// test for blocks in content, but usually called after the blocks have been convereted
 		if (
-			tribe_get_option( 'toggle_blocks_editor' )
-			&& (
-				has_blocks( $post->ID )
-				|| ! $template_overwrite->has_classic_editor( $post->ID )
-			)
+			has_blocks( $content )
+			|| false !== strpos( (string) $content, 'tribe-block' )
```
Comment (sc0ttkclark) [inline src/Tribe/RSVP.php]: Will it ever not have blocks in $content but still have tribe-block?

Comment (Camwyn) [reply src/Tribe/RSVP.php]: I don't believe so - that is the class we put on blocks.

Comment (Camwyn) [reply src/Tribe/RSVP.php]: that class does appear in other pages (I'm looking at you AR) but I'm not seeing any issues in my tests.

Comment (Camwyn) [reply src/Tribe/RSVP.php]: By changing it to `tribe-block ` (with a space at the end) we're specifically targeting blocks and avoiding tripping on things like `tribe-block__rsvp__ticket tribe-block__rsvp__ticket--inactive`

**Pull request merged**